GoogleMeetup/src/main/webapp/script.js

```diff
@@ -458,4 +465,54 @@ function averageLatLongs() {
     };
 
     return midpoint;
+}
+
+
+// Does a nearby search (10km) of midpoint for selected destination type
```

Comment:
The parameter is 2000 (meters, I presume), so perhaps this comment needs updating?